server/plugin.go

```diff
@@ -260,6 +277,20 @@ func (p *Plugin) handleDMs(bot *Bot, channel *model.Channel, postingUser *model.
 		return err
 	}
 
+	if post.RootId == "" {
+		p.track(evUserStartedConversation, map[string]any{
+			"user_actual_id":   postingUser.Id,
+			"bot_id":           bot.mmBot.UserId,
+			"bot_service_type": bot.cfg.Service.Type,
+		})
+	} else {
+		p.track(evContextualInterrogation, map[string]any{
```

Comment:
Can we change this to `evContinuedConversation` as this is more than contextual interrogation.

Reply:
Done